democracy/migrations/0012_add_hearing_slug.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.2 on 2016-05-11 11:06
+from __future__ import unicode_literals
+
+import autoslug.fields
+from django.db import migrations
+
+
+def backwards(apps, schema_editor):
+    pass
+
+
+def forwards(apps, schema_editor):
+    """
+    Add slugs to existing Hearings.
+    """
+    hearing_model = apps.get_model('democracy', 'Hearing')
+    for hearing in hearing_model.objects.all():
+
+        # populate slugs from ids to avoid collisions which aren't easy to deal with here
```

Comment:
Using IDs is superior to random ones, because most of them should contain slugs, and they are hopefully unique also :) The point was just that normal slug from title cannot be used in this as that can lead to collisions.